### PR TITLE
[IMP] sales: online payment confirmation doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3459500&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### FWP up to `master`

### CODE REVIEW

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 1]
```diff
@@ -1,53 +1,87 @@
...
+.. note::
+   Odoo will **only** offer payment options on the :guilabel:`Validate Order` pop-up window that
+   have been published and configured on the :guilabel:`Payment Providers` page.
 
-.. tip::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have to choose when it comes to payment.
+Once the customer selects their desired method of payment, they will click the :guilabel:`Pay`
+button on the pop-up window, thus officially confirming the order. Odoo will instantly notify the
+salesperson attached to the order when the customer confirms the order with an online payment.
```
Comment (jero-odoo) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:82]: Notified how? (I assume via internal notifications or emails depending on settings but might be nice to have a screenshot showing how they are informed).

**Review by jero-odoo — APPROVED** [review]
@ksc-odoo great work, just a few comments 👍

**Comment by ksc-odoo** [comment]
Thanks for the super-helpful review, @jero-odoo --- looks like this one is ready for a review from the Materials team now (@hojo-odoo , @Felicious , or @brse-odoo ) 👍

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 2]
```diff
@@ -1,53 +1,93 @@
...
+   feature option, located in the :guilabel:`Online confirmation` field of the quotation template
+   form.
+
+   .. image:: get_paid_to_validate/online-payment-option-quotation-template.png
+      :align: center
+      :alt: The online payment setting on quotation template forms in Odoo Sales.
+
+Beneath the :guilabel:`Online Payment` option on the *Sales* :guilabel:`Settings` page, there's a
+:guilabel:`Default Quotation Validity` field. In this field, there's the option to add a specific
+number of days for quotations to remain valid by default.
```
Comment (Felicious) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:42]: I think this section can be moved up to immediately follow the instruction to check the "Online Payment" feature box, so people don't need to jump back to the Sales > configuration > settings page after going to the quotation

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 3]
```diff
@@ -1,53 +1,93 @@
...
+
+Payment providers
+=================
+
+After checking the box beside the :guilabel:`Online Payment` feature on the :guilabel:`Settings`
+page, a link to the :guilabel:`Payment Providers` appears beneath it.
```
Comment (Felicious) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:48]:
```suggestion
After activating the :guilabel:`Online Payment` feature, a link to configure :guilabel:`Payment
Providers` appears beneath it.
```

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 4]
```diff
@@ -1,53 +1,93 @@
...
+
+.. seealso::
+   - :doc:`/applications/finance/payment_providers`
 
 Register a payment
 ==================
 
-After opening quotations from their received email, your customers will have different
-possibilities to make their online payments. For example:
+After opening quotations in their customer portal, and customers click :guilabel:`Accept & Pay` to
```
Comment (Felicious) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:63]:
```suggestion
After opening quotations in their customer portal, customers can click :guilabel:`Accept & Pay` to
```

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 5]
```diff
@@ -1,53 +1,93 @@
...
+.. note::
+   Odoo will **only** offer payment options on the :guilabel:`Validate Order` pop-up window that
+   have been published and configured on the :guilabel:`Payment Providers` page.
 
-.. tip::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have to choose when it comes to payment.
+Once the customer selects their desired method of payment, they will click the :guilabel:`Pay`
+button on the pop-up window, thus officially confirming the order. Odoo will instantly notify the
+salesperson attached to the order when the customer confirms the order with an online payment.
```
Comment (Felicious) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:84]:
```suggestion
button on the pop-up window to confirm the order. Odoo instantly notifies the assigned salesperson
upon order confirmation with an online payment.
```
Optional suggestion for brevity (:

**Review by Felicious — APPROVED** [review]
Amazing work with this doc, @ksc-odoo :blush: 

The instructions are easy to follow and the writing flows super well, with a great balance of admonition blocks and images to keep the read entertaining. 

I just had a few suggestions for brevity, feel free to pick and choose which you'd like to keep!

**Comment by ksc-odoo** [comment]
Thank you for the always-helpful feedback @Felicious -- I made all your suggested edits, and believe this PR is now ready for Final Review @odoo/us-doc-review 👍

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 6]
```diff
@@ -1,53 +1,91 @@
...
+
+Clicking that link reveals a separate :guilabel:`Payment Providers` page, in which a large variety
+of payment providers can be enabled, customized, and published.
+
+.. image:: get_paid_to_validate/payment-providers-page.png
+   :align: center
+   :alt: Payment providers page in Odoo Sales.
+
+.. seealso::
+   - :doc:`/applications/finance/payment_providers`
```
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:56]: Update doc paths to be relative
```suggestion
   :doc:`../../../finance/payment_providers`
```

File: content/applications/sales/sales/send_quotations/get_paid_to_validate.rst [hunk 7]
```diff
@@ -1,53 +1,91 @@
...
+upon order confirmation with an online payment.
+
+.. image:: get_paid_to_validate/payment-confirmation-notification-chatter.png
+   :align: center
+   :alt: Sample of notification that appears in the chatter when an online payment is made.
 
 .. seealso::
-   - :doc:`quote_template`
+   - :doc:`/applications/sales/sales/send_quotations/quote_template`
+   - :doc:`/applications/sales/sales/send_quotations/get_signature_to_validate`
```
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/get_paid_to_validate.rst:90]:
```suggestion
   - :doc:`quote_template`
   - :doc:`get_signature_to_validate`
   - :doc:`../../../finance/payment_providers`
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo this PR looks good to me, nice work! I have a very small change that I will push up to move this PR forward. As well as updating the commit message since it is a bit too many characters. Thank you!

**Comment by samueljlieber** [comment]
@StraubCreative this PR is good to go 👍

**Comment by StraubCreative** [comment]
@robodoo r+

**Review by StraubCreative — APPROVED** [review]

**Pull request closed without merging**